azure-pipelines.yml

```diff
@@ -0,0 +1,13 @@
+jobs:
```

Comment:
let me know if I need to remove this file. merge conflict

Reply:
@jgw96 Should this be removed?

Reply:
Yep that should be removed, i already removed it from dev which explains the merge conflict.